app/Http/Controllers/ProfileController.php

```diff
@@ -74,4 +75,12 @@ public function profile_image(Request $request)
     
         return redirect()->back()->with('success', 'Profile picture URL updated successfully!');
     }
+
+    public function all_chirpers()
```

Comment:
rename this method 'index'
[see here](https://github.com/laravel/ideas/issues/1195)
[and here](https://laravel.com/docs/10.x/controllers#resource-controllers)
These are guidelines, not requirements, but you should try to stick to them for the most part